dts/bindings/led/ti,lp5562.yaml

```diff
@@ -25,3 +25,10 @@ properties:
     default: 175
     description: Output current of white channel in 0.1 mA (0-25.5 mA)
                  Default value is the power-on default. Valid range = 0 - 255
+
+  en-gpios:
```

Comment:
Is that a dual-purpose pin as on the LP5569 chip ? If not then I wonder if it would not be better to add a fixed regulator in the devicetree configuration of your board. See https://github.com/zephyrproject-rtos/zephyr/pull/76713#pullrequestreview-2233320574 for reference.

Also I think you should rename `en-gpios` into `enable-gpios`. And this addition should be mentioned into the release notes.

And finally, I think you can squash this commit with the next one.

Reply:
The datasheet says "VCC/EN" so it only has one purpose. The fixed regulator is actually how I solved it for our project. (It gave us the flexibility of enabling/disabling multiple devices with one output).

But in this implementation the enable-gpios are optional so a user can choose, right?

Reply:
@simonguinot updated the name, but i don't know where to add it in release notes, Could you point me to the location?

and yes `enable-gpios` is single purpose for 5562, and is optional.

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/release-notes-4.0.rst

Reply:
I can see it after rebasing, and updated it.
Thanks!

Reply:
> The datasheet says "VCC/EN" so it only has one purpose. The fixed regulator is actually how I solved it for our project. (It gave us the flexibility of enabling/disabling multiple devices with one output).
> 
> But in this implementation the enable-gpios are optional so a user can choose, right?

Well, it looks weird to me to duplicate the same code in every driver to handle power through a GPIO.